static/redfish/v1/JsonSchemas/OemLogEntry/index.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This file is better to be renamed (to handle Validator properly)
```
static/redfish/v1/JsonSchemas/OemLogEntry/OemLogEntry.json
``

Reply:
will fix